cleaning.py

```diff
@@ -0,0 +1,49 @@
+from langdetect import detect 
+from scrapper import main as scrap
+import pandas as pd
+
+def clean(elementos, label):
+    df = pd.DataFrame(elementos, columns=['Text'])
+    df['Label'] = label
+    df.drop_duplicates(subset='Text', inplace=True)
+
+    filter_df = df[df['Text'].str.len() > 20] 
+    filter_lan = df[filter_df['Text'].apply(lambda x: detect(x) == "en")]  
+    return filter_lan
```

Comment:
Esto está peliagudo de entender, porque `filter_df` y `filter_lan` no son filtros, son el dataframe filtrado, creo que sería mejor expresarlo en un par de líneas más.
```suggestion
    filter_length = df['Text'].str.len() > 20
    df = df[filter_length] 
    filter_lang = df['Text'].apply(lambda x: detect(x) == "en")
    df = df[filter_lang]
    return df
```